axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/analytic/AnalyticLineModelFromEmployeeServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.axelor.apps.businessproject.service.analytic;
+
+import com.axelor.apps.account.db.AnalyticMoveLine;
+import com.axelor.apps.account.db.repo.AnalyticMoveLineRepository;
+import com.axelor.apps.hr.db.Employee;
+import com.axelor.apps.supplychain.model.AnalyticLineModel;
+import com.google.inject.Inject;
+import java.util.ArrayList;
+
+public class AnalyticLineModelFromEmployeeServiceImpl
+    implements AnalyticLineModelFromEmployeeService {
+
+  protected AnalyticMoveLineRepository analyticMoveLineRepo;
+
+  @Inject
+  public AnalyticLineModelFromEmployeeServiceImpl(AnalyticMoveLineRepository analyticMoveLineRepo) {
+    this.analyticMoveLineRepo = analyticMoveLineRepo;
+  }
+
+  @Override
+  public void copyAnalyticsDataFromEmployee(
```

Comment:
Can AnalyticLineEmployeeModel  works for this purpose ?

Reply:
Not really sure how I should do it